src/bonding-curves/GDACurve.sol

```diff
@@ -151,95 +119,85 @@ contract GDACurve is ICurve, CurveErrorCodes {
         external
         view
         override
-        returns (
-            Error error,
-            uint128 newSpotPrice,
-            uint128 newDelta,
-            uint256 outputValue,
-            uint256 protocolFee
-        )
+        returns (Error error, uint128 newSpotPrice, uint128 newDelta, uint256 outputValue, uint256 protocolFee)
     {
-        // NOTE: we assume delta is > 1, as checked by validateDelta()
-
         // We only calculate changes for buying 1 or more NFTs
         if (numItems == 0) {
             return (Error.INVALID_NUMITEMS, 0, 0, 0, 0);
         }
 
-        uint256 invAlpha;
-        {
-            (uint256 alpha, , ) = _parseDelta(delta);
-            invAlpha = FixedPointMathLib.WAD.fdiv(alpha, FixedPointMathLib.WAD);
-        }
-        uint256 invAlphaPowN = invAlpha.fpow(numItems, FixedPointMathLib.WAD);
-
-        // For an exponential curve, the spot price is divided by alpha for each item sold
-        // safe to convert newSpotPrice directly into uint128 since we know newSpotPrice <= spotPrice
-        // and spotPrice <= type(uint128).max
-        newSpotPrice = uint128(
-            uint256(spotPrice).fmul(invAlphaPowN, FixedPointMathLib.WAD)
-        );
-        if (newSpotPrice < MIN_PRICE) {
-            newSpotPrice = uint128(MIN_PRICE);
-        }
+        uint256 spotPrice_ = uint256(spotPrice);
+        (uint256 alpha,,) = _parseDelta(delta);
+        // TODO: this value may overflow, should we cap the value?
```

Comment:
cc @sudo-owen

Reply:
True, but we will still need a cap on the time for this direction otherwise the price will be too large

Reply:
We want to limit the `boostFactor` right, not necessarily time elapsed? e.g. if lambda is very very small, e.g. 0.5% a day, then even after 100 days, the price could still be valid

Reply:
Yeah, that's what I meant

Reply:
Makes sense. Thoughts on 1/100 and 100x being sane ceilings for the buy and sell side?

Reply:
It's hard to tell - I currently have it set to 1024 in my latest PR, but I still need to think about the potential effects of other values